### PR TITLE
add ActiveSupport::Cache::RedisCacheStore#info

### PR DESCRIPTION
**Pull request by ritikesh**
Added a `info` method to `RedisCacheStore` similar to its counterpart's `MemCacheStore#stats`

### CODE REVIEW

File: activesupport/lib/active_support/cache/redis_cache_store.rb [hunk 1]
```diff
@@ -319,6 +319,11 @@ def clear(options = nil)
         end
       end
 
+      # Get info from redis servers.
+      def info
```
Comment (rafaelfranca) [inline activesupport/lib/active_support/cache/redis_cache_store.rb:323]: Let's keep the name consistent and call this method `stats`.

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: activesupport/lib/active_support/cache/redis_cache_store.rb (hunk 1 above)
Comment (ritikesh) [reply activesupport/lib/active_support/cache/redis_cache_store.rb]: @rafaelfranca - done.

**Comment by rafaelfranca** [comment]
Looks good to me. Can you add a CHANGELOG entry?

**Comment by ritikesh** [comment]
> Looks good to me. Can you add a CHANGELOG entry?

Done. Thanks.

**Pull request merged**